gulpfile.js

```diff
@@ -58,11 +59,11 @@ gulp.task('pretty', function () {
     .pipe(gulp.dest('./build/'));
 });
 
-// Build CSS
-gulp.task('buildCss', function() {
-  gulp.src(path.css)
-  //.pipe(concat('html5-skin.css'))
-  .pipe(gulp.dest('build'));
+// Build Sass
+gulp.task('sass', function () {
+  gulp.src(path.sass)
+    .pipe(sass({outputStyle: 'compressed'}).on('error', sass.logError))
```

Comment:
We need to have a 2 versions of min and un-minified of css files.
please read the guideline on naming convention